src/Symfony/Component/DependencyInjection/Tests/Fixtures/Utils/ToBeExcluded.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Tests\Fixtures\Utils;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class ToBeExcluded
```

Comment:
It'd be great to allow this feature to work even if the attribute class doesn't not exist.
Can you remove this one and make the feature still work?